src/main/java/com/onarandombox/MultiverseCore/world/SimpleMVWorldManager.java

```diff
@@ -1001,24 +985,24 @@ public FileConfiguration getConfigWorlds() {
     /**
      * {@inheritDoc}
      */
-	@Override
-	public boolean hasUnloadedWorld(String name, boolean includeLoaded) {
-		if (getMVWorld(name) != null) {
-			return includeLoaded;
-		}
-		for (Map.Entry<String, WorldProperties> entry : this.worldsFromTheConfig.entrySet()) {
-			if (name.equals(entry.getKey()) || name.equals(entry.getValue().getAlias())) {
-				return true;
-			}
-		}
-		return false;
-	}
+    @Override
+    public boolean hasUnloadedWorld(String name, boolean includeLoaded) {
+        if (getMVWorld(name) != null) {
+            return includeLoaded;
+        }
+        for (Map.Entry<String, WorldProperties> entry : this.worldsFromTheConfig.entrySet()) {
+            if (name.equals(entry.getKey()) || name.equals(entry.getValue().getAlias())) {
+                return true;
+            }
+        }
+        return false;
+    }
 
     /**
      * {@inheritDoc}
      */
     @Override
-	public Collection<String> getPotentialWorlds() {
+    public Collection<String> getPotentialWorlds() {
```

Comment:
I think all these formatting issue should be a separate pr, I think your IDE prob auto fixed the formatting Bcu you edited the file lol

Reply:
Yep, that was the case lol